vars/piperExecuteBin.groovy

```diff
@@ -33,9 +51,20 @@ void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, fai
             Map config = readJSON(text: sh(returnStdout: true, script: "./piper getConfig --contextConfig --stepMetadata '.pipeline/tmp/${metadataFile}'"))
             echo "Config: ${config}"
 
+            String piperCommandLine = "./piper ${stepName}"
+
+            // Extend command line for config files to align with Jenkins side
+            if (customDefaultsString) {
+                piperCommandLine += " --defaultConfig ${customDefaultsString}"
+            }
+            if (script.commonPipelineEnvironment.configurationFile
+                && script.commonPipelineEnvironment.configurationFile != '.pipeline/config.yaml') {
+                piperCommandLine += " --customConfig \"$script.commonPipelineEnvironment.configurationFile\""
```

Comment:
Maybe you should apply to bash utils here instead of manually escaping it.

Reply:
Done.